src/modules/Security.rb

```diff
@@ -620,8 +611,24 @@ def write_polkit_settings
       end
     end
 
-    # Ensures that file permissions and PolicyKit privileges are applied
-    def apply_new_settings
+    # Apply sysctl changes into the running system.
+    #
+    # When the sysctl YaST config file conflicts with other sysctl files, it
+    # applies the changes system wide. Otherwise it only applies the changes
+    # from the sysctl YaST config file.
+    def apply_sysctl_changes
+      args = sysctl_config.conflict? ? ["--system"] : ["-p", CFA::Sysctl::PATH]
```

Comment:
@jreidinger ok, so as discussed in the mailing list, I can omit this verification and call always `/usr/sbin/sysctl --system`

@imobachgs , what do you think?

Reply:
yes, it is fine for me. So behavior is consistent.